### PR TITLE
RELEASE-CHECKLIST: fix a typo

### PR DESCRIPTION
**Pull request by chenhanxiao**
s/cabdidate/candidate

Signed-off-by: Chen Hanxiao chenhanxiao@cn.fujitsu.com

### CODE REVIEW

File: project/RELEASE-CHECKLIST.md [hunk 1]
```diff
@@ -217,7 +217,7 @@ We recommend announcing the release candidate on:
 - In a comment on the pull request to notify subscribed people on GitHub
 - The [docker-dev](https://groups.google.com/forum/#!forum/docker-dev) group
 - The [docker-maintainers](https://groups.google.com/a/dockerproject.org/forum/#!forum/maintainers) group
-- Any social media that can get bring some attention to the release cabdidate
+- Any social media that can get bring some attention to the release candidate
```
Comment (RickWieman) [inline project/RELEASE-CHECKLIST.md]: I think the word 'get' should be omitted.

Comment (chenhanxiao) [inline project/RELEASE-CHECKLIST.md]: Yep, will update.

**Comment by RickWieman** [comment]
LGTM (not a maintainer)

**Comment by crosbymichael** [comment]
Thanks @chenhanxiao @RickWieman 

LGTM

**Pull request merged**